### DISCUSSION TITLE
composite video with 2 wires

### DISCUSSION THREAD

**Discussion by netpipe** [fdivitto/FabGL#174] [Ideas]
https://playground.arduino.cc/Main/TVout/ was wondering if we can support this too please.

**Comment by netpipe** [comment]
possibly using headphone jack instead

**Comment by fdivitto** [comment]
is that black/white?

**Comment by netpipe** [comment]
just the yellow wire, the red and white are for audio.

**Reply by fdivitto** [reply]
I mean, the output is Color or Black/White?

**Reply by netpipe** [reply]
yes color on only the yellow composite jack. the  yellow red and white RCA cables for older televisions and VCR's.

**Reply by netpipe** [reply]
there are also some headphone jacks with stereo that can do video and audio too with 3 or 4 different layers on the end. they sell the cables on ebay and i had one for my myvu crystal HMD. just a suggestion for future hardware if we end up needing it more.

**Comment by fdivitto** [comment]
ok

**Comment by fdivitto** [comment]
please follow this:

[https://github.com/fdivitto/FabGL/discussions/192](https://github.com/fdivitto/FabGL/discussions/192)
